### DISCUSSION TITLE
Unable to install Chia for 1.2.6 and 1.2.7 (setup crash)

### DISCUSSION THREAD

**Discussion by JBDoc777** [Chia-Network/chia-blockchain#8548] [Support]
Tried looking online and uninstalling old 1.2.3 (unable to clean registry)

Encountered error for both setup versions. Problem started when my 1.2.3 wasnt connecting and sync-ing decided to update to 1.2.6. Still found old 1.2.3 in windows apps but unable to uninstall. (used Windows 10 64 bitt App uninstall troubleshooter no registry issues reported)
When checked error log, there were references to my discord folder??

Tried to clean system for virus and also setup with admin privileges. 
Who can i send the error log securely?

Thanks

**Comment by loppefaaret** [comment]
https://keybase.io/team/chia_network.public that is our official chat, you can DM @pyl on there, and securely send the log to me

**Comment by BrandtH22** [comment]
@JBDoc777 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved.
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
